RocketChat/Rocket.Chat
# fix: Inconsistent Markdown Formatting in Custom Status Field

https://rocketchat.atlassian.net/browse/SUP-537




## Proposed changes (including videos or screenshots)

https://github.com/RocketChat/Rocket.Chat/assets/85572761/808e9770-590d-45dc-a33d-519049c8b8f3

## Issue(s)

Closes #32544
Closes #32081 

## Steps to test or reproduce

1. Navigate to the Profile section.
2. Enter a Custom Status using * (asterisk) and _ (underscore) signs.
3. Observe that the text enclosed within * (asterisk) marks will be displayed in bold





## Further comments

For now, I have successfully tested this with input containing * (asterisk) and _ (underscore) signs. It would be better if I could have more test cases.
